### PR TITLE
Added xkcd_downloader and timymodule

### PR DESCRIPTION
**Pull request by shreydan**
- Uses python 3
- New xkcd_downloader.py
- New timymodule.py
- Also updated readme

### CODE REVIEW

File: xkcd_downloader.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	Please move the file to another directory to avoid messing with the folder structure.
+"""
+
+import requests
+from lxml import html
+import urllib.request
+import os
+
+# opens xkcd.com
+try:
```
Comment (ankit167) [inline xkcd_downloader.py]: Good to organise the code below inside a function or inside __name__ == '__main__'

Comment (shreydan) [inline xkcd_downloader.py]: Should I delete this pull request and create a new one with the new changes?

Comment (shreydan) [inline xkcd_downloader.py]: Okay, I'll update the files soon. No need to delete. (I'm new to Git)

**Comment by shreydan** [comment]
@ankit167 Updated as requested

**Comment by geekcomputers** [comment]
Thank you for you contribution to the repo and the community

**Pull request merged**